src/ipelib/empirical_efield/efield_ipe.f

```diff
@@ -1537,6 +1464,7 @@ subroutine interp_poten( pot_highlats, pot_highlat, pot_midlat,
         if (j2 < 0) j2 = 0              ! Tomoko's fix - j2 >= 0
 	wrk1 = pot_midlat(ilon,j1)
 	wrk2 = pot_highlats(ilon,j2)
+!        write(iulog,*) 'pot_all ',ilon,hb1,hb2,nmlath -ibnd,tw
```

Comment:
unnecessary comment

Reply:
in old code and found it was helpful for my coding, so that for others